app/views/shops/index.html.erb

```diff
@@ -3,10 +3,10 @@
   <% @shops.each do |shop| %>
     <li>
       <%= link_to shop.name, company_shop_path(@company, shop) %>
-    </li>
-  <% end %>
-  </ul>
-</br>
-<%= link_to "Create" , new_company_shop_path%>
-</br>
-<%= link_to "Back to the company", companies_path %>
+      <% end %>
+  	</li>
```

Comment:
why did you change this? please be careful when resolving conflicts. much more careful. you broke a lot of places